libraries/AP_GPS/AP_GPS_NOVA.cpp

```diff
@@ -226,7 +226,7 @@ AP_GPS_NOVA::process_message(void)
                 case 34: // narrow float
                 case 48: // l1 int
                 case 50: // narrow int
-                    state.status = AP_GPS::GPS_OK_FIX_3D_RTK;
+                    state.status = AP_GPS::GPS_OK_FIX_3D_RTK_FLOAT;
```

Comment:
Probably, the approach taken was that if it was unclear (and I considered the number of options here unclear) that I should only flag it as RTK_FLOAT.